r/tests/testthat/test-dplyr-funcs-string.R

```diff
@@ -467,6 +467,18 @@ test_that("strsplit and str_split", {
   )
 })
 
+test_that("strrep", {
+  df <- tibble(x = c("foo1", " \tB a R\n", "!apACHe aRroW!"))
+  for (times in 0:8L) {
+    compare_dplyr_binding(
+      .input %>%
+        mutate(x = strrep(x, times)) %>%
+        collect(),
+      df
+    )
+  }
+})
+
```

Comment:
Adds a test for the `str_dup()` binding I suggested above. Also FYI you don't need the `L` after `8` because the `:` operator in R always creates integer vectors when its operands are whole numbers.
```suggestion
test_that("strrep, str_dup", {
  df <- tibble(x = c("foo1", " \tB a R\n", "!apACHe aRroW!"))
  for (times in 0:8) {
    compare_dplyr_binding(
      .input %>%
        mutate(x = strrep(x, times)) %>%
        collect(),
      df
    )
    compare_dplyr_binding(
      .input %>%
        mutate(x = str_dup(x, times)) %>%
        collect(),
      df
    )
  }
})
```